maven-java/kalix-maven-archetype-event-sourced-entity/src/main/resources/archetype-resources/docker-compose.yml

```diff
@@ -15,9 +15,9 @@ services:
       USER_FUNCTION_HOST: ${USER_FUNCTION_HOST:-host.docker.internal}
       USER_FUNCTION_PORT: ${USER_FUNCTION_PORT:-8080}
       # Comment to enable ACL check in dev-mode (see https://docs.kalix.io/services/using-acls.html#_local_development_with_acls)
-      ACL_ENABLED: 'false'
+      ACL_ENABLED: "false"
       # Uncomment to enable advanced view features locally (note: disabled in deployed services by default)
-      #VIEW_FEATURES_ALL: "on"
+      #VIEW_FEATURES_ALL: "true"
```

Comment:
It turns out that templates already had the settings uncommented.

So, I'm just replaced it by `true` to align with other vars.

Reply:
Ok, nice.. it did cross my mind previously. Cool 👍